topoflow/components/glacier_energy_balance.py

```diff
@@ -0,0 +1,778 @@
+"""
+This file defines an "energy balance" glacier melt and snowmelt component and related
+functions.  It inherits from the glacier "base class" in
+"glacier_base.py".
+"""
+#-----------------------------------------------------------------------
+#
+#  Copyright (c) 2001-2023, Scott D. Peckham
+#
+#  Sep 2023. Checked sign in initialize_cold_content().
+#            Separate function: update_cold_content().
+#            Moved initialize_cold_content() back to base class.
+#            Renamed T0 to avoid conflict w/ degree-day method.
+#            Removed separate version of initialize_computed_vars().
+#  Aug 2023. Removed trailing space in 'J kg-1 K-1 '.
+#            Added missing "vol_swe" in initialize_computed_vars.
+#
```

Comment:
I am not sure if this log is still copy of snow, but maybe add or revise this log??